lib/bot/cmds/thanks.js

```diff
@@ -35,7 +35,8 @@ const thanksCommands = {
 
     const namesList = mentions.reduce((userList, mention) => {
       const toUser = mention.screenName.toLowerCase();
-      if (toUser !== fromUser && userList.indexOf(toUser) === -1) {
+
+      if (toUser !== fromUser && toUser !== 'all'  && userList.indexOf(toUser) === -1) {
```

Comment:
@abhisekp , is it good now? https://github.com/FreeCodeCamp/camperbot/pull/78/commits/2e431ed2f3da662ae38c5dd690376b09450897aa